### ISSUE TITLE
Show panels only in 2 of 3 displays

### ISSUE THREAD

**Issue by joaoewald** [spin83/multi-monitors-add-on#93]
Hello, I would like to know if it's possible to choose which monitor I want to add the panels.
I have 3 and need to show only in 2.

**Comment by felagund** [comment]
Yes, settings per monitor instead on either alll/primary monitor woudl be great!
